responders/MailIncidentStatus/mailincidentstatus.py

```diff
@@ -0,0 +1,272 @@
+        else:
+            date_str = "Unknown"
+        updatedAt_row = ('updatedAt',date_str)
+        updatedBy_row = ('updatedBy',self.get_param("data.updatedBy"))
+        table_rows = [case_row,title_row,severity_row,tlp_row,status_row,description_row,tasks_row,startDate_row,createdAt_row,createdBy_row,updatedAt_row,updatedBy_row]
+        
+        ## Custom fields
+        cust_fields = self.get_param("data.customFields",None,"Error loading customFields")
+        cust_field_rows = []
+        for item in sorted(cust_fields):
+            # value of item is dictionary with one element
+            # sample: "scope-accounts-compromised":{"string":"makr"}
+            cust_value_type = next(iter(cust_fields.get(item)))
+            if cust_value_type == "date":
+                date_int = (cust_fields.get(item)).get(cust_value_type)
+                if date_int:
+                    date_str = (datetime.datetime.fromtimestamp(date_int / 1e3)).strftime('%m/%d/%Y %H:%M')
+                else:
+                    date_str = "Date not set"
+                cust_value_str = date_str
+            else:
+                cust_value_str = str((cust_fields.get(item)).get(cust_value_type))
+            cust_field_rows.append((item,cust_value_str))
+        
+        table_rows.extend(cust_field_rows)
+        body = self.create_HTMLTable(table_rows)
+        return body
+
+    def get_HTMLSeverityString(self,severity):
+        if severity == 1:
+            return "<p style=\"color:DeepSkyBlue\">Low</p>"
+        elif severity == 2:
+            return "<p style=\"color:Orange\">Medium</p>"
+        elif severity == 3:
+            return "<p style=\"color:Red\">High</p>"
+        elif severity == 4:
+            return "<p style=\"color:DarkRed\">Critical</p>" 
+        else: 
+            return "Severtiy mapping failed"
+
+    def create_HTMLTable(self,two_tuple_list):
+        """
+            Create a HTML tabel out of a list of string tuples. In the frist colum the first element of the tuple is representated, in the second column the second element of the tuple is present.
+        """
+
+        html = "<!DOCTYPE html>\n"
+        html += "<html>\n"
+        html += "<head>\n"
+        # meta definitions
+        html += "<meta charset=\"UTF-8\"/><meta http-equiv=\"Content-Type\" content=\"text/html\"; charset=\"utf-8\"/>\n"
+        # styles
+        html += "<style>" + self.get_param('config.mail_html_style_tag_content',None,"Error loading config 'config.mail_html_style_tag_content'") + "</style>\n"
+        #html += self.get_HTMLStyle()
+        html += "</head>\n\n"
+        html += "<body>\n"
+        html += "<table width=\"100%\" cellpadding=\"0\" cellspacing=\"0\" border=\"0\" bgcolor=\"#FFFFFF\" align=\"center\">\n"
+        html += "<colgroup><col class=\"first\"/><col class=\"second\"/></colgroup>\n"
+        html += "<tr><td colspan=\"2\">Incident Status Report</td></tr>\n"
+        for i in two_tuple_list:
```

Comment:
updatedBy can be None: ('updatedBy', None) which will trigger error:
`html += i[1] TypeError: can only concatenate str (not "NoneType") to str`